PlayBooks/Official/Version-1/Large-Company/en/OpenChain Playbook - Large Company.md

```diff
@@ -0,0 +1,192 @@
+
+Produce a report enumerating the products identified.
+
+## Step 3
+
+Review the processes and policies identified by the survey, and compare them for consistency. Compare the processes and policies against OpenChain to identify gaps. Produce a report identifying the findings of the survey and comparisons, indicating where there are policy differences, process differences, and gaps in relation to OpenChain.
+
+## Step 4
+
+Revise the scope of the initial project. It may be decided that a specific subset of the surveyed target area will be the initial scope for implementation. Define the timeline for remediating the policies and processes used within the revised scope.
+
+Define the initial needs for the OSPO: structure, staffing, tooling, budget. Determine where the OSPO will live within the company.
+
+Gain buy-in from the senior sponsor, and gain approval for the budget and staffing.
+
+Hire the OSPO leader and initial staff.
+
+## Step 5
+
+The OSPO produces revised policies and processes, and works with the product teams for the revised scope to implement the new processes.
+
+The OSPO works to update the existing onboarding training, to reflect new processes.
+
+NOTE: It is recommended that these documents be as short and as simple as possible. Unless the company already has sophisticated processes in place the company can build the compliance program over time. It is far better to have something basic that can be implemented soon rather than something perfect that will take too long to implement.
+
+## Step 6
+
+The OSPO holds meetings during the midpoint and the endpoint of the timelines for implementation in each department. The goals are:
+
+1. 	To support and encourage each department in the execution of the plan with the combined experience of the OSPO members.
+2.	 To keep the management team informed of progress and to adjust timescales as necessary to ensure success.
+
+# Reviewing Our Results
+
+When each department reports their activity is complete, the OSPO has a special workshop to review the results against OpenChain ISO/IEC 5230:2020 self-certification questionnaire. 
+
+1. 	If the results match the questionnaire the OSPO team members create a report to management explaining that self-certification conformance is complete. 
+2.	 If the results do not match the questionnaire the OSPO team members return to Step 6 and help the relevant department complete their work.
+
+# Periodic Reassessment
+
+The OSPO schedules a review of the program every 18 months to ensure it maintains conformance with OpenChain ISO/IEC 5230:2020. This also meets one of the requirements of the standard.
+
+The OSPO also schedules a review of current company processes every six months to help ensure everything is optimized. This review is based on reports from individual project managers about how quick and effective current processes are, and which current processes are causing challenges for the teams.
+ 
```

Comment:
This will also be quite variable.  My team, for example, has a process activity that continually keeps an eye on the processes and ear to the ground for possible changes, and also we survey the customers for satisfaction and suggestions.  My suggestion:  Remove "every six months" and just say "regularly".